README.md

```diff
@@ -4,11 +4,8 @@ Publisher is the primary content design app for GOV.UK. It provides the user int
 entering all the key editorial formats and an API so other apps (primarily frontend) can
```

Comment:
```suggestion
```

Optional: all of this seems wrong: it's neither the key app, nor has any particular API that I'm aware of. The only thing that's useful in this paragraph appears to be the pseudonym sentence.